test/distributed/_tensor/test_math_ops.py

```diff
@@ -9,13 +9,13 @@
 from torch.distributed._tensor.placement_types import Replicate, Shard
 from torch.testing._internal.common_utils import run_tests
 from torch.testing._internal.distributed._tensor.common_dtensor import (
-    DTensorTestBase,
+    DTensorOpTestBase,
     skip_unless_torch_gpu,
     with_comms,
 )
 
 
-class DistMathOpsTest(DTensorTestBase):
+class DistMathOpsTest(DTensorOpTestBase):
     @with_comms
```

Comment:
I don't think we need the `with_comms` wrapper when switching to OpTestBase, please fix those